namespaces/live.cloud-platform.service.justice.gov.uk/data-platform-app-probation-recruitment-app-dev/00-namespace.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: data-platform-app-probation-recruitment-app-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HQ"
+    cloud-platform.justice.gov.uk/slack-channel: "people-analytics"
+    cloud-platform.justice.gov.uk/application: "Probation Workforce Vacancies"
+    cloud-platform.justice.gov.uk/owner: "workforce experience, planning and insight division: "
```

Comment:
There is no email id mentioned. Is it still on progress? It would be useful to have a working communication email